elements/bulbs-header/bulbs-header.js

```diff
@@ -0,0 +1,65 @@
+import { registerElement, BulbsHTMLElement } from 'bulbs-elements/register';
+import './bulbs-header-masthead';
+import './bulbs-header-responsive-nav';
+import './bulbs-header.scss';
+
+import { InViewMonitor } from 'bulbs-elements/util';
+
+class BulbsHeader extends BulbsHTMLElement {
+  createdCallback () {
+    this.mastheads = this.getElementsByTagName('bulbs-header-masthead');
+    this.responsiveNavs = this.getElementsByTagName('bulbs-header-responsive-nav');
```

Comment:
Is there ever going to be more than one of either of these?  Wondering if we should do the `[0]` index look up here and eliminate those prop accessors below